docs/usage-tui.md

```diff
@@ -73,7 +73,10 @@ and these functions in more detail.
 ### General syntax
 
 To begin with, all `Knit` functions accept keyword arguments, some of which may be optional.
-Optional arguments are marked with "`?`" suffix in the help. To specify a keyword argument, write
+Optional arguments are marked with "`?`" suffix in the help. If an
+argument can be specified many times (including zero), it's marked
+with "`*`" suffix. If it can be specified many times, but at least
+once, it's marked with "`+`" suffix. To specify a keyword argument, write
```

Comment:
I wanted to avoid this regex BS in an _user_ manual :unamused:

Reply:
I think it's ok for TUI users :man_shrugging: